### DISCUSSION TITLE
RosettaDock-5.0: ERROR: Some of the Rosetta processes crashed!

### DISCUSSION THREAD

**Discussion by Ryu-yuki-res** [RosettaCommons/rosetta#155] [ROSIE]
Hi. I tried docking two proteins using RosettaDock-5.0 in the ROSIE service. 
I dropped the PDB file to the server. This PDB data includes two data (chain A and B), referring as below.
I wrote "A_B" in the "Docking partners:".
But, the analysis crashed. 
Would you happen to know how to solve this problem?

<img width="1060" alt="スクリーンショット 2024-09-09 18 58 40" src="https://github.com/user-attachments/assets/42e98c43-b876-4cf1-91ab-1859fa217076">
<img width="1126" alt="スクリーンショット 2024-09-09 18 55 37" src="https://github.com/user-attachments/assets/13179f70-75d8-4c04-bd79-cd0d16345b4f">
[ROSETTA_CRASH.log](https://github.com/user-attachments/files/16928815/ROSETTA_CRASH.log)

Regards, 
Zabu

**Comment by alexssfaioga** [comment]
Excuse me, have you found the cause of the problem? I am experiencing the same issue

**Comment by roccomoretti** [comment]
@Ryu-yuki-res 's crash is related to disulfides. Rosetta is trying to figure out what residues are disulfide bonded to which other residues, and is finding a residue which it thinks should be disulfide bonded, but which doesn't look to be connected to anything. -- I'm not sure I could say more from the information provided.

**Comment by mahaouarab-hub** [comment]
same I have the same error, and xhenever I vizualise it in ChimeraX I see the S-S disulfide bonded already, so I just don't know what to do?

**Reply by roccomoretti** [reply]
If you are getting a ROSETTA_CRASH.log file with the lines 

```
[FILE]: src/core/scoring/disulfides/FullatomDisulfideEnergyContainer.cc
[LINE]: 618
[START_MESSAGE]
[ ERROR ] UtilityExitException
```

It's an error related to figuring out disulfides, and the same as above. If your ROSETTA_CRASH.log has different lines, it's a different issue. In that case, please open a new Discussion, and include the ROSETTA_CRASH.log information you're getting for your docking run.
